src/components/tabs/themes/tabs.base.scss

```diff
@@ -136,6 +136,7 @@ igc-icon-button::part(base) {
     position: relative;
     grid-row: 2;
     grid-column: 1 / -1;
+    text-align: initial;
```

Comment:
I saw that the body content also gets centered, so I think it's better to have `text-align: initial` on the host(igx-tabs)

Reply:
Or if you can't add it on the host, you can also add it on the body, but you have to do it for the body in Angular as well